src/api/one/profiler/AsyncProfiler.java

```diff
@@ -208,8 +208,17 @@ private void filterThread(Thread thread, boolean enable) {
         }
     }
 
+    public void setContextId(long contextId) {
```

Comment:
Can you add javadoc here explaining that `0` value is reserved for 'no-context' and as such should not be used to refer to a valid context?